### PR TITLE
Migrated TokenStore.RemoveAllTokensByType to Sync by default (#11345)

### PR DESCRIPTION
**Pull request by ollykel**
#### Summary
* changed TokenStore.RemoveAllTokensByType to return *model.AppError directly

* changed test coverage accordingly in store/storetest/mocks

#### Ticket Link
Fixes [issue #11345](https://github.com/mattermost/mattermost-server/issues/11345)

### CODE REVIEW

File: store/sqlstore/tokens_store.go [hunk 1]
```diff
@@ -74,11 +74,15 @@ func (s SqlTokenStore) Cleanup() {
...
 
-func (s SqlTokenStore) RemoveAllTokensByType(tokenType string) store.StoreChannel {
-	return store.Do(func(result *store.StoreResult) {
-		if _, err := s.GetMaster().Exec("DELETE FROM Tokens WHERE Type = :TokenType", map[string]interface{}{"TokenType": tokenType}); err != nil {
-			result.Err = model.NewAppError("SqlTokenStore.RemoveAllTokensByType", "store.sql_recover.remove_all_tokens_by_type.app_error", nil, err.Error(), http.StatusInternalServerError)
-			return
-		}
-	})
+func (s SqlTokenStore) RemoveAllTokensByType(tokenType string) *model.AppError {
+	if _, err := s.GetMaster().Exec(
```
Comment (jespino) [inline store/sqlstore/tokens_store.go]: I would prefer to respect the previous indentation approach (Just one line for the return and for the Exec).

Comment (ollykel) [reply store/sqlstore/tokens_store.go]: Point taken, amended the commit

**Comment by jespino** [comment]
@ollykel Please avoid the `push -f` and rebases, we prefer merges and addintional commits because is easier to us to keep track of the changes (actually, at the end we always do "Squash and Merge", so the final result will be the same). Now I have to review the whole PR again. Not a big deal because is a small one, but would be good to follow the policy. Thanks!

**Comment by cpoile** [comment]
@jespino There's one failing check -- do you want me to merge anyway? (And for the future if that `serverless-ops/github` check fails -- what does it mean?)

**Comment by jespino** [comment]
@cpoile that normally means that the branch is too far from master, and is better to wait until the contributor merge master into the branch.

**Comment by jespino** [comment]
@ollykel do you mind to merge master into the branch?

**Pull request merged**